tests/hypothesis_strategies/targets.py

```diff
@@ -5,28 +5,28 @@
 from baybe.targets.enum import TargetMode
 from baybe.targets.numerical import _VALID_TRANSFORMATIONS, NumericalTarget
 
-from .utils import interval
+from .utils import intervals as st_intervals
 
 target_name = st.text(min_size=1)
 """A strategy that generates target names."""
 
 
 @st.composite
-def numerical_target(draw: st.DrawFn):
+def numerical_targets(draw: st.DrawFn, intervals=None):
     """Generate :class:`baybe.targets.numerical.NumericalTarget`."""
     name = draw(target_name)
     mode = draw(st.sampled_from(TargetMode))
-    bounds = draw(
-        interval(
+    if intervals is None:
```

Comment:
Why not `if not intervals`? Is this just a question of style? (Just asking out of curiosity)

Reply:
No, rather because there is a difference between `if x` and `if x is not None` or respectively `if not x` and `if x is None`: it's the exact same issue that we had in Tobi's error fix (other things will also evaluate falsy, such as empty lists etc). Here, the situation is even less clear because of the missing type hint, which makes it even unspecified what types `intervals` can even take except for `None`. If we placed something more precise, such as `interval: Optional[Interval] = None`, then it ultimately boils down to the question if objects of type `Interval` can evaluate to false or not.